### PR TITLE
Avoid flooded log files

### PR DESCRIPTION
**Pull request by ggruening**
This fixes issue https://github.com/kereis/traefik-certs-dumper/issues/90

### CODE REVIEW

File: bin/dump.sh [hunk 1]
```diff
@@ -412,5 +412,7 @@ while true; do
     if [[ $? -eq 0 ]]; then
       dump
     fi
+  else
+    sleep 10
```
Comment (kereis) [inline bin/dump.sh:416]: Should we maybe make this configurable via environment variables?

File: bin/dump.sh [hunk 2]
```diff
@@ -407,6 +407,10 @@ mkdir -p ${workdir}
 dump
 
 while true; do
-  inotifywait -qq -e modify "${acme_file_path}"
-  dump
+  if [[ -f ${acme_file_path} ]]; then
```
Comment (kereis) [inline bin/dump.sh:410]: If `${acme_file_path}` is not a file, we leave it as it is completely silent. Shouldn't we `exit 1` if that's the case?

**Review by kereis — CHANGES_REQUESTED** [review]
Hi,

Thanks again for your contribution! 👍 
I left some review comments. :)

Cheers

File: bin/dump.sh (hunk 2 above)
Comment (ggruening) [reply bin/dump.sh:410]: For me, that's a _design decision_ I can't really make. If we do it as you suggested, we end the container. This is also likely to happen completely silently. E.g. someone would reconfigure the `traefik` container and temporarily rename the output file, our container would die. 
In my suggestion, it survives and waits for a proper file. I wanted to keep the status that has been followed so far. 
I don't know which is better. What's your opinion? I'm afraid it's up to you as the author to decide. :-)

File: bin/dump.sh (hunk 1 above)
Comment (ggruening) [reply bin/dump.sh:416]: We can do that, even if I don't think there are many users who want to adjust something here... But a variable is not a problem. Is `$retry_interval` a good name for it? Or do we need it more specific?

**Comment by ggruening** [comment]
@kereis , I commented your review comments. ;) There are two decisions left for me.

File: bin/dump.sh (hunk 1 above)
Comment (kereis) [reply bin/dump.sh:416]: Hm, I guess if someone needs that feature, then they should simply create an issue/PR.

File: bin/dump.sh (hunk 2 above)
Comment (kereis) [reply bin/dump.sh:410]: I haven't really thought about this one and to be honest, I don't know what would be best practice. Maybe we leave it as it is for now.

**Review by kereis — CHANGES_REQUESTED** [review]
Got one more suggestion before merging. :)

**Review by kereis — APPROVED** [review]
LGTM. Thanks again. :)

**Pull request merged**